BreakingChanges.txt

```diff
@@ -1,3 +1,8 @@
+Tracking Breaking Changes since 0.6.1
+  - Change the TransferException format, the TransferException with following 2 error code won't expose inner exception details (e.g. RequestId) in TransferException message any more
```

Comment:
"the TransferException with following 2 error code" => "the TransferExceptions with following 2 error code" or "the TransferException with following 2 error codes". Not quite sure about this.

Reply:
fixed